ci/build_wheel_cpp.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Copyright (c) 2023-2024, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+package_name="libkvikio"
+package_dir="python/libkvikio"
+
+source rapids-configure-sccache
+source rapids-date-string
+
+rapids-generate-version > ./VERSION
+
+cd "${package_dir}"
+
+python -m pip install wheel
+python -m pip wheel . -w dist -vvv --no-deps --disable-pip-version-check
```

Comment:
```suggestion
# libkvikio is a header-only C++ library with no Python code, so
# it is entirely platform-agnostic. We cannot use auditwheel for
# retagging since it has no extension modules, so we use `wheel`
# directly instead.
python -m pip wheel . -w dist -vvv --no-deps --disable-pip-version-check
```